functions/linear_algebra/correlation_angles.stan

```diff
@@ -12,7 +12,7 @@
    */
 matrix angle2chol (matrix angle_mat){
   int N = rows(angle_mat);
-  matrix[N, N] inv_mat = identity_matrix(N);
+  matrix[N, N] inv_mat;
```

Comment:
Let's add the doxygen style format.

Also include yourself under the copyright!